app/viewmodels/helpers/messages/ViewMessageHelper.scala

```diff
@@ -247,7 +262,7 @@ class ViewMessageHelper @Inject()(
           link(appConfig.emcsTfeAlertOrRejectionUrl(ern, arc), "messages.IE704.IE819.fixError.link", id = Some("submit-a-new-alert-rejection")),
           Html(messages("messages.IE704.IE819.fixError.text.2"))
         ))))
-      case "IE813" if isPortalSubmission =>
+      case "IE813" if draftMovementExists =>
         Seq(p()(HtmlFormat.fill(Seq(
           Html(messages("messages.IE704.IE813.fixError.text")),
           link(appConfig.emcsTfeChangeDestinationUrl(ern, arc), "messages.IE704.IE813.fixError.link", id = Some("submit-change-destination"), withFullStop = true)
```

Comment:
This will need to be reworked as part of the future alignment work. Currently, this code is unreachable as the backend will only check the CaM user answers. I believe UCD are looking at alternate content for IE813